pytransform3d/test/_utils.py

```diff
@@ -0,0 +1,30 @@
+import numpy as np
```

Comment:
Not sure why this works, but test/_utils.py should be importable as test is not a subpackage of pytransform3d (and also should not be one). Otherwise we would have to put an __init__.py here. I would just move this helper function to the test file.

Reply:
Done